content/pages/agenda.md

```diff
@@ -0,0 +1,106 @@
+---
+title: "Agenda"
+layout: "agenda"
+url: "agenda"
+---
+
+
+
+<style>
+td {vertical-align:top;}
+.agenda {
+border-width:2px;
+border-style:solid;
+border-color:black;
+border-collapse: collapse;
+width:60%;
+}
+
+th, td {
+  padding: 10px;
+}
+
+.agenda td {
+border-width:1px;
+border-style:solid;
+border-color:black;
+}
+
+.agendaLink {color: blue; text-decoration: none;}
+.agendaLink:hover {text-decoration: underline;}
+.agendaLink:active {color: black;}
+.agendaLink:visited {color: purple;}
+
+.timecontainer {width:20%;}
+.trainingcontainer {width:20%;}
+.descriptioncontainer {width:60%px;}
+
+</style>
+
+<!-- </style> -->
+
+<h1>Communication</h1>
+  <h4>Join us on <a href="https://join.slack.com/t/cascadiarconf/shared_invite/zt-1lu53059t-GAxQtzrwQhmo7BXE7YfC8w" target="blank_">Slack</a></h4>
+  <br>
+
+## June 21, 2024
+<br>
+
+#### 9:00-9:10 Introductions
+
+#### 9:10-9:40 Keynote 
+
+* [Deepsha Menghani](/2024/keynote/deepsha_menghani): Harnessing the power of Gen AI with RAGs
+
+#### 9:40-9:55 Break
+
+#### 9:55-10:55 R-evolution: Articifial intelligence & R
+
+* [Nikhita Damaraju](/2024/regular/nikhita_damaraju): R-tificial intelligence: A guide to using R for ML  
+* [Emily Kraschel](/2024/regular/emily_kraschel): R Workflows in Azure Machine Learning for Athletic Data Analysis
```

Comment:
Emily's image not rendering in subpage

Reply:
We don't have an image for Emily

Reply:
Ah ok. In the past, we've then just put the hex logo
`../../../../img/logo/logo_2024/logo_2024.png`